packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -294,12 +298,32 @@ const buildAnswers = ({
     }
 }
 
+const pkgSingleLocaleData = {
+    l10n: {
+        supportedCurrencies: ['USD'],
+        defaultCurrency: 'USD',
+        supportedLocales: [
+            {
+                id: 'en-US',
+                preferredCurrency: 'USD'
+            }
+        ],
+        defaultLocale: 'en-US'
+    }
+}
+
+const PWAKitConfigSingleLocaleData = {
+    url: {
+        locale: 'none'
+    }
+}
```

Comment:
How about extracting these config-related objects into files in the folder 'packages/pwa-kit-create-app/assets/'? The ticket asks about creating files in that asset folder. It may be better to group all the files together in the same location.

Reply:
Good call! I didn't know that we added an `/assets` folder to the generator and I missed that implementation detail. 

Done, now the configuration objects are files inside the assets folder.